### DISCUSSION TITLE
Upgarde Issue

### DISCUSSION THREAD

**Discussion by sc-hero** [filamentphp/filament#7845] [General]
In composer.json

{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The skeleton application for the Laravel framework.",
    "keywords": ["laravel", "framework"],
    "license": "MIT",
    "require": {
        "php": "^8.2.8",
        "guzzlehttp/guzzle": "^7.2",
        "lara-zeus/bolt": "^1.2",
        "laravel/framework": "^10.10",
        "laravel/sanctum": "^3.2",
        "laravel/tinker": "^2.8"
    },
    "require-dev": {
        "fakerphp/faker": "^1.9.1",
        "filament/upgrade": "^3.0-stable",
        "laravel/pint": "^1.0",
        "laravel/sail": "^1.18",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.1",
        "spatie/laravel-ignition": "^2.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}

Output::
  Problem 1
    - Root composer.json requires filament/filament ^3.0-stable, found filament/filament[v3.0.0, ..., 3.x-dev] but these were not loaded, likely because it conflicts with another require.
  Problem 2
    - filament/spatie-laravel-translatable-plugin v2.17.52 requires filament/filament v2.17.52 -> found filament/filament[v2.17.52] but it conflicts with your root composer.json require (^3.0-stable).
    - lara-zeus/bolt 1.2.12 requires filament/spatie-laravel-translatable-plugin ^2.0 -> satisfiable by filament/spatie-laravel-translatable-plugin[v2.17.52].
    - lara-zeus/bolt is locked to version 1.2.12 and an update of this package was not requested.
    
    ** How to upgrade filament v2 to v3?

**Comment by danharrin** [comment]
The error tells you, Bolt is out of date
